python/tvm/relay/frontend/pytorch.py

```diff
@@ -1607,6 +1607,12 @@ def matmul(self, inputs, input_types):
                 return _op.reshape(output, [*a_shape[:-2], a_shape[-2], b_shape[-1]])
             return output
 
+        # Reshape a or b into a 2 dimensional tensor
```

Comment:
I feel merging them to the above if-branch could be clearer:

```
if both a and b > 2:
  Batch matmul
elif a > 2:
  Reshape a
elif b > 2:
  Reshape b
```

Reply:
Done. To avoid generating redundant reshape/transpose op, I handle the dim of b independently.